voluptuous.py

```diff
@@ -699,6 +699,8 @@ def validate_sequence(path, data):
                         invalid = e
                 else:
                     if len(invalid.path) <= len(index_path):
+                        if invalid is not None:
```

Comment:
This can never be `None` due to the previous `if len(invalid.path) ...` but it seems like it should be checked somewhere.